microsoft-integration/admin-tool/src/main/resources/Messages_eu.properties

```diff
@@ -128,6 +128,9 @@ config.SYNCH\:ADD_USER_TO_CHANNEL=When user is added to a group, add to related
 config.SYNCH\:REMOVE_USER_FROM_CHANNEL=When user is removed from a group, remove from related Channel(s) (requires a valid and forced Group-Channel synchronization)
 config.SYNCH\:CREATE_INVITATION=If Microsoft user does not exist, create guest invitation.
 
+config.onedrive_config=OneDrive Config
```

Comment:
@Pablosanroman would you mind translating these strings? Thanks

Reply:
config.onedrive_config=OneDrive konfigurazioa

Reply:
Thanks! updated